
#input: command + target

# task: get(fetch & carry task) , move, stop, search
string task

#temporary use here: kitchen, milk, [x, y, theta]
string content   ## to be deprecated and replaced with parameters

string[] parameters  # not in use at the moment. 

# Or other task information

# geometry_msgs/Pose2D userPose
string userPose

---

# 0: successful. 1: unsuccessful. 2: other
int32 result

# generated sessionId
int32 sessionId

#if other case
string description
